docs/integrations/microsoft/sso-saml.mdx

```diff
@@ -0,0 +1,92 @@
+---
+description: Use SAML with Microsoft Entra ID to provide SSO to the ngrok Dashboard
+---
+
+# Microsoft Entra ID SSO (SAML)
+
+---
+
+:::tip TL;DR
+
+To secure access to ngrok Dashboard using Microsoft Entra ID Single Sign-On with SAML:
+
+1. [Configure Entra ID SSO](#configure-entra)
+1. [Configure ngrok](#configure-ngrok)
+1. [Test access to ngrok with Entra ID SSO](#test-sso)
+
+:::
+
+This article details how to configure Microsoft Entra ID as the primary Identity Provider for the ngrok Dashboard.
+By integrating Entra ID SSO with ngrok, you can:
+
+- **Restrict access to the ngrok Dashboard** only to users authenticated via Entra ID
+- **Use Entra ID's Dashboard to facilitate access to the ngrok app**.
+
+## Requirements
+
+To configure the ngrok Dashboard with Microsoft Entra ID, you must have:
```

Comment:
```suggestion
To configure the ngrok Dashboard with Microsoft Entra Verified ID, you must have:
```

Reply:
The Product itself is Microsoft Entra ID. Verified ID is included in Entra ID. In Azure the service is referred to as Entra ID.

I also believe that Verified ID is used for JWT.

Verified ID is currently included with any Microsoft Entra ID subscription, including Microsoft Entra ID Free, at no additional cost. https://www.microsoft.com/en-us/security/business/identity-access/microsoft-entra-verified-id#tabx67298482964d423da017e0eae7ba14b3